jflex/src/main/java/jflex/Options.java

```diff
@@ -46,6 +47,8 @@
    * If true, dot (.) metachar matches [^\n] instead of [^\r\n\u000B\u000C\u0085\u2028\u2029]|"\r\n"
    */
   public static boolean legacy_dot;
+  /** The encoding to use for input files. */
```

Comment:
What about the output?

Reply:
You are right, it'd be better to apply this encoding to both, so that things are consistent. I don't think it's worth doing two separate options, because if somebody wants the output file in a different encoding to the one specified, they can still recode it, as long as it's clear what the output encoding was